pkg/result/filter.go

```diff
@@ -69,11 +72,29 @@ func FilterResult(ctx context.Context, result *types.Result, ignoreConf IgnoreCo
 	if opt.PolicyFile != "" {
 		var err error
 		var ignored int
-		filteredVulns, filteredMisconfs, ignored, filteredSecrets, filteredLicenses, err = applyPolicy(ctx, filteredVulns, filteredMisconfs, filteredSecrets, filteredLicenses, opt.PolicyFile)
+
+		// If the PolicyFile option is a dir find and apply rego files in it
+		var policyFiles []string
+		fi, err := os.Stat(opt.PolicyFile)
 		if err != nil {
-			return xerrors.Errorf("failed to apply the policy: %w", err)
+			return xerrors.Errorf("file %q stat error: %w", opt.PolicyFile, err)
```

Comment:
I kind of like both of these errors, maybe something that says both like:
```suggestion
			return xerrors.Errorf("failed to apply the policy: path %q, error: %w", opt.PolicyFile, err)
```
perhaps?

Reply:
Makes sense 👍